moveit_core/collision_detection_fcl/include/moveit/collision_detection_fcl/collision_common.h

```diff
@@ -55,6 +55,8 @@
 #include <memory>
 #include <set>
 
+#include "rclcpp/rclcpp.hpp"
```

Comment:
If this is only used inside the .cpp file I would move it there.

Reply:
534f02660d153111486c3751806ac252b88edeb7